src/neural/blas/transforms.h

```diff
@@ -63,15 +62,10 @@ class Transforms {
                            const std::vector<float>& biases,
                            std::vector<float>& output, bool apply_relu = false);
 
-  template <size_t spatial_size>
   static void Batchnorm(size_t channels, std::vector<float>& data,
```

Comment:
I  kind I liked the const float* formulation, and at least, it was consistent with the 3rd argument.

Reply:
That whole function left me feeling icky. We're using C++, there shouldn't be any pointer arithmetic anyways. blegh. Would you be satisfied if I changed the `eltwise` one to be a vector of floats as well?